packages/react-core/src/components/TextInputGroup/TextInputGroupMain.tsx

```diff
@@ -19,25 +37,35 @@ export const TextInputGroupMain: React.FunctionComponent<TextInputGroupMainProps
   children,
   className,
   icon,
+  type,
   onChange,
+  onFocus,
+  onBlur,
   'aria-label': ariaLabel,
```

Comment:
Do you want to set a default value for this aria label? I saw you deleted it a few lines down, but you can set it inline here.

Reply:
The default aria label value is just moved down to line 70 along with the default type value, if you'd rather me do the defaults inline here though I'm happy to do so.

Reply:
oh - i hadn't noticed that! @tlabaj do you prefer one way or the other?

Reply:
We don't do this anywhere else. Neither is wrong,.  For reviewing purposes, it maybe more intuitive to go with what we have elsewhere.